platyPS.schema.md

```diff
@@ -23,7 +23,10 @@ It closely resembles output of `Get-Help`.
 
         ## SYNTAX
         // for each parameter set
+            ### Parameter Set Name
+            ```
             {{Output of Get-Command -Syntax}}
```

Comment:
In comparing the new schema to TechNet:
One thing that happens a lot is the syntax block gets very long, and can run far off the page 
(an example of this is [Add-Computer](https://technet.microsoft.com/en-us/library/hh849798.aspx)).
Is there a way consider wrapping or some kind of GFM truncation? 
I don't think this is a concern for MVP or a blocker, but it does provide a somewhat hard to read user experience.